src/style.css

```diff
@@ -21,6 +21,8 @@ body {
   overflow: hidden;
   position: absolute;
   width: 100%;
+  min-width: 1440px;
```

Comment:
Can we do 1200px? That seems to work everywhere based on my own testing and it's more comfortable for me.